docs/source/transcoding.md

```diff
@@ -255,7 +255,18 @@ You can run the above command on any number of machines that you would like to d
 
 ## GPU transcoding
 
-When you setup split transcoding for your orchestrator, you can run individual transcoders on not only CPUs, but GPUs as well. The GPUs that transcoders run on can be dedicated purely to transcoding or can also be mining cryptocurrencies simultaneously. See [this guide](https://github.com/mk-livepeer/bot-miner-setup) for instructions on how to setup transcoders on GPUs.
+When you setup split transcoding for your orchestrator, you can run individual transcoders on not only CPUs, but GPUs as well. The GPUs that transcoders run on can be dedicated purely to transcoding or can also be mining cryptocurrencies simultaneously. See [this guide](https://github.com/livepeer/bot-miner-setup) for instructions on how to setup transcoders on GPUs.
+
+**Running a standalone Transcoder with GPUs**
+
+```
+$ livepeer -transcoder -orchAddr <ORCH_SERVICE_URI> -orchSecret <ORCH_SECRET> -nvidia <GPU IDS (comma separated)>
+```
+
+`<ORCH_SECRET>` should be the secret defined by your orchestrator. `<ORCH_SERVICE_URI>` should be the publicly accessible `serviceURI` that your orchestrator registered on-chain.
+
+You can choose which GPUs you want to use by their PCIe ID. To get the PCIe IDs of your GPUs you can use `nvidia-smi`. 
```

Comment:
This information is actually already included in the [go-livepeer GPU docs](https://github.com/livepeer/go-livepeer/blob/master/doc/gpu.md#gpu-support). What do you think about just linking to those docs in the last sentence of the paragraph above?

Example:

"""
When you setup split transcoding...

See the [GPU support docs](https://github.com/livepeer/go-livepeer/blob/master/doc/gpu.md#gpu-support) for more information on transcoding on GPUs and [this guide](https://github.com/livepeer/bot-miner-setup) for instructions on transcoding and mining cryptocurrencies simultaneously on GPUs.
""

Reply:
I think our readthedocs is the central place where our documentation should live. One area of feedback I've gotten is that the docs are too scattered.

Reply:
Agreed that we should have a central place to point people to. But, I'd prefer not to duplicate information that already exists as keeping the information in sync across multiple locations will be a burden which is why I suggest linking to the [GPU docs](https://github.com/livepeer/go-livepeer/blob/master/doc/gpu.md#gpu-support) here. 

Open to suggestions on maintaining a central place to point people to while also making it easy for docs to be updated as the code is updated (which is the benefit of the `docs` directory in go-livepeer). One possibility is for the files in the `docs` directory in go-livepeer to be automatically rendered in RTD so that the information can be updated in that directory, but the information would be viewed in RTD.